Makefile

```diff
@@ -28,12 +29,15 @@ update-k8s-conf:
 teardown: teardown-k8s-utils teardown-env teardown-secrets teardown-remote-state
 
 teardown-remote-state:
-	pushd terraform/bootstrap/remote-state; \
-	terraform destroy -auto-approve -var "environment=$(ENV)";
+	export AWS_PAGER=''; \
+	aws s3 rb s3://<% .Name %>-$(ENV)-terraform-state --force; \
+	aws dynamodb delete-table --table-name <% .Name %>-$(ENV)-terraform-state-locks;
 
 teardown-secrets:
-	pushd terraform/bootstrap/secrets; \
-	terraform destroy -auto-approve;
+	export AWS_PAGER=''; \
+	aws secretsmanager  list-secrets --query "SecretList[?Tags[?Key=='project' && Value=='<% .Name %>']].[Name]" | jq  '.[] [0]' | xargs aws secretsmanager delete-secret --secret-id; \
+	aws iam delete-access-key --user-name <% .Name %>-ci-user --access-key-id $(shell aws iam list-access-keys --user-name <% .Name %>-ci-user --query "AccessKeyMetadata[0].AccessKeyId" | sed 's/"//g'); \
```

Comment:
they actually dont cascade delete entities it seems, i get the following error without deleting the keys
 >An error occurred (DeleteConflict) when calling the DeleteUser operation: Cannot delete entity, must delete access keys first.

Reply:
Okay, one more thing to try.. The `--access-key-id` param to `delete-access-key` takes a regex. Could you just say `-access-key-id .*`?

Reply:
their man page is lying i suspect, it may support regex but it needs to go through a bunch of validation that conforms the keyID's schema 🤦 

with `.*?` 
>Parameter validation failed:
Invalid length for parameter AccessKeyId, value: 3, valid range: 16-inf

i tried to make it 16 character or above with `\w\w\w\w\w\w\w\w[\w]+` then it gave me 
>An error occurred (ValidationError) when calling the DeleteAccessKey operation: The specified value for accessKeyId is invalid. It must contain only alphanumeric characters.

Reply:
Sheesh. Oh well, thanks for trying!